src/client/websocket/packets/handlers/GuildCreate.js

```diff
@@ -5,18 +5,19 @@ class GuildCreateHandler extends AbstractHandler {
   async handle(packet) {
     const client = this.packetManager.client;
     const data = packet.d;
+    data.shard = packet.shard || client.shard;
```

Comment:
If `data.shard` is client.shard, then it will not have `.status`

And is there any chance of packet.shard to be null?